src/applications/gi-sandbox/containers/ComparePage.jsx

```diff
@@ -221,6 +222,7 @@ export function ComparePage({
         />
       )}
       <div className="content-wrapper">
+        <GIBillHeaderInfo />
```

Comment:
No this should exist outside of any pre-existing element so as to preserve dom order